subsys/mgmt/hawkbit/shell.c

```diff
@@ -78,14 +79,86 @@ static int cmd_info(const struct shell *sh, size_t argc, char *argv)
 	shell_fprintf(sh, SHELL_NORMAL, "Unique device id: %s\n", device_id);
 	shell_fprintf(sh, SHELL_NORMAL, "Firmware Version: %s\n",
 		      firmware_version);
+	shell_fprintf(sh, SHELL_NORMAL, "Server address: %s\n", hawkbit_get_server_addr());
+	shell_fprintf(sh, SHELL_NORMAL, "Server port: %d\n", hawkbit_get_server_port());
+	shell_fprintf(sh, SHELL_NORMAL, "DDI security token: %s\n",
+		      (IS_ENABLED(CONFIG_HAWKBIT_DDI_NO_SECURITY)
+			       ? "<disabled>"
+			       : hawkbit_get_ddi_security_token()));
 
 	return 0;
 }
 
+static int cmd_init(const struct shell *sh, size_t argc, char *argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	shell_fprintf(sh, SHELL_INFO, "Init hawkBit ...\n");
+
+	hawkbit_init();
```

Comment:
risky if user calls this more than once

Reply:
`hawkbit_init()` now includes a check if it has already been initialised